src/createRoom.ts

```diff
@@ -233,7 +233,7 @@ export default async function createRoom(client: MatrixClient, opts: IOpts): Pro
         createOpts.initial_state.push(makeSpaceParentEvent(opts.parentSpace, true));
         if (!opts.historyVisibility) {
             opts.historyVisibility =
-                createOpts.preset === Preset.PublicChat ? HistoryVisibility.WorldReadable : HistoryVisibility.Invited;
+                createOpts.preset === Preset.PublicChat ? HistoryVisibility.WorldReadable : HistoryVisibility.Shared;
```

Comment:
Setting aside for now whether this is a valid change (see  https://github.com/vector-im/element-meta/issues/377#issuecomment-1688212895):

If Element-Android and Element-iOS just follow the server's defaults in this situation, should we not also do so here?